app/ide-desktop/lib/dashboard/src/components/Devtools/EnsoDevtools.tsx

```diff
@@ -37,17 +41,19 @@ const PaywallDevtoolsContext = React.createContext<{
 })
 
 /**
- * Props for the {@link PaywallDevtools} component.
+ * Props for the {@link EnsoDevtools} component.
```

Comment:
avoid using product name in code. consider either `AppDevtools` or `DashboardDevtools` or even simply `Devtools`

Reply:
Enso Devtools sounds like a brand, don't you think? In UI it's also "Enso devtools", so I think keeping naming makes sense here. 
![CleanShot 2024-07-10 at 16 03 59@2x](https://github.com/enso-org/enso/assets/61194245/e17823e2-988a-4790-b1aa-5b83a518b03c)